Adyen/Assets/Generated/LocalizationKey.swift

```diff
@@ -1,7 +1,8 @@
 //
-// Copyright (c) 2024 Adyen N.V.
+// Copyright (c) 2023 Adyen N.V.
```

Comment:
Do we have year hardcoded somewhere?

Reply:
This is interesting. Let me check on this.

Reply:
In my local copy now it is showing changes again but the 2024 one 🤔 . I did nothing